AK/Function.h

```diff
@@ -56,6 +56,14 @@ class Function<Out(In...)> {
     using FunctionType = Out(In...);
     using ReturnType = Out;
 
+#ifdef KERNEL
+    constexpr static auto AccommodateExcessiveAlignmentRequirements = false;
+    constexpr static size_t ExcessiveAlignmentThreshold = alignof(void*);
+#else
+    constexpr static auto AccommodateExcessiveAlignmentRequirements = false;
+    constexpr static size_t ExcessiveAlignmentThreshold = 16;
```

Comment:
Should this maybe be 8 on 32-bit? (I'm not sure!)

Reply:
Doesn't really matter tbh, more than 16 is absurd on all platforms I know of, which is what I went with
It's just a diagnostic tool, the value only matters with the AccommodateBlahBlah flag set to false, and then it just gives you an error asking you to review your captures.